demo-app/src/ClipPathPreviewForm.js

```diff
@@ -0,0 +1,67 @@
+import React from "react";
+import Form from "react-bootstrap/Form";
+import "./ClipPathPreviewForm.css";
+
+function ClipPathPreviewForm(props) {
+    return(
+        <Form>
+            <Form.Group>
```

Comment:
validate the required fields.

name, formula

Reply:
Done